### DISCUSSION TITLE
When are you planning to release Django 4.* compatible version

### DISCUSSION THREAD

**Discussion by lukaszla** [celery/django-celery-beat#547] [Q&A]
My project uses your great library but its now stopping us to migrate to Django 4+ as it explicitly restricts its version to <4. 
I see in master you've some code to support latest Django so my question is when are you planning to release it?

**Comment by SALIMA534** [comment]
Thank you, @lukaszla for raising this question, because we are also running into same issue. That is, we have Django 4+ version on our development server. When we install _django-celery-beat_ Django 4+ version was forcefully downgraded to Django 3.2.13, which is really surprising. We really need Django 4+ version for our app and it became a blocker now. I appreciate if Django 4+ version support is added to this _django-celery-beat_ package.

**Reply by auvipy** [reply]
by the end of this month. can you please try master branch and see which tests are failing please?

**Reply by lukaszla** [reply]
I can't use anything except pypi released packages, sorry. Company policy :(

**Reply by auvipy** [reply]
not even in dev environment?

**Reply by SALIMA534** [reply]
Thank you @auvipy . Yes, we also can not use the master branch because we installed the package in our development server and if new package is available with fix we will try it.

**Comment by auvipy** [comment]
released
